### PR TITLE
Implemented Chat Screen

### PR DESCRIPTION
**Pull request by ashishiiit01**
Implemented Chat Screen UI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,24 +8,25 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/drawer": "^5.11.5",
     "@react-navigation/native": "^5.9.0",
     "expo": "~40.0.0",
     "expo-status-bar": "~1.0.3",
     "native-base": "^3.0.0-next.11",
+    "prop-types": "^15.7.2",
```
Comment (theankurkedia) [inline package.json:17]: I don't think this is needed.

File: utils/colorTheme.js [hunk 2]
```diff
@@ -3,5 +3,10 @@ export const newColorTheme = {
     100: '#005A86',
     200: '#DFF3FF',
     300: '#CBDEEB',
+    400: '#DFF3FF',
```
Comment (theankurkedia) [inline utils/colorTheme.js]: Can we define color properly. Like give them a name or something. Ideally, this is used to add different shades to one color.

Comment (ashishiiit01) [reply utils/colorTheme.js]: Completed this

File: package.json (hunk 1 above)
Comment (ashishiiit01) [reply package.json:17]: PropType is needed as I was passing props to other component.

File: utils/colorTheme.js [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
   bg: {
-    100: '#005A86',
-    200: '#DFF3FF',
-    300: '#CBDEEB',
-    400: '#DFF3FF',
-    500: '#F2F2F2',
-    600: '#008DD6',
-    700: '#60BFED',
-    800: '#61C0EE',
+    header: '#005A86',
```
Comment (theankurkedia) [inline utils/colorTheme.js]: This is not what I meant. You have to define colors outside of `bg`.

File: package.json (hunk 1 above)
Comment (theankurkedia) [reply package.json:17]: No, it is not. Read about it.

**Pull request merged**